portal-impl/src/com/liferay/portal/internal/servlet/MainServlet.java

```diff
@@ -342,8 +342,7 @@ public void init() throws ServletException {
 
 		try {
 			String xml = StreamUtil.toString(
-				servletContext.getResourceAsStream(
-					"/WEB-INF/shielded-container-web.xml"));
+				servletContext.getResourceAsStream("/WEB-INF/web.xml"));
```

Comment:
@jorgediaz-lr The xml content is handled in `_checkWebSettings(String)` method. In this method, there is logic to initialize the `I18nServlet` and `I18nFilter`:

```
		I18nServlet.setLanguageIds(root);

		I18nFilter.setLanguageIds(I18nServlet.getLanguageIds());
```

If you simply revert the commit, this initialization logic will not be read, because the configs are in `shielded-container-web.xml`.

I think the proper way to handle this is to split the method, using separate logic for reading session timeout config (which is from web.xml) and reading `I18nServlet` definitions (which is from shielded-container-web.xml).

Reply:
oops, I didn't notice that we were calling `I18nServlet.setLanguageIds(root);` there

Yes, It is necessary to split the method.

@dantewang I am closing this PR, I will resend it.